hangry_games/src/models/tribute.rs

```diff
@@ -122,11 +123,20 @@ impl Tribute {
         let mut brain = tribute.brain.clone();
 
         // Get nearby tributes
-        let area_tributes = self.area().unwrap().tributes();
+        let area_tributes = area.tributes();
         let living_tributes = area_tributes.iter().filter(|t| t.is_alive && t.health > 0 && t.game_id == self.game_id);
         let nearby_tributes: Vec<TributeActor> = living_tributes.clone().map(|t| TributeActor::from(t.clone())).collect();
         let nearby_targets: Vec<Tribute> = living_tributes.into_iter().cloned().collect();
 
+        // If the tribute is in a closed area, move them.
+        let game = get_game_by_id(connection, self.game_id.unwrap());
+        if let Ok(game) = game {
+            if game.closed_areas.expect("Couldn't get closed areas").contains(&Some(area.id)) {
+                self.move_tribute(connection, tribute.clone());

```

Comment:
Might add a comment/TK for logic that eliminates a player if their area is closing and their movement is < 25 (`move_tribute` : L253)